src/Microsoft.DotNet.Interactive.Formatting.Tests/HtmlFormatterTests.cs

```diff
@@ -326,6 +326,70 @@ public void It_formats_non_generic_dictionaries_that_arent_generic_as_tables_wit
                           "<table><thead><tr><th><i>key</i></th><th>TypeName</th><th>Id</th></tr></thead><tbody><tr><td>first</td><td>entity one</td><td>123</td></tr><tr><td>second</td><td>entity two</td><td>456</td></tr></tbody></table>");
             }
 
+            [Fact]
+            public void It_formats_DateTime_correctly()
+            {
+                var date1 = DateTime.Now;
+
+                var html = date1.ToDisplayString("text/html");
+
+                html.Should().BeEquivalentHtmlTo(
+                    $"<span>{date1.ToString("u")}</span>");
+            }
+
+            [Fact]
+            public void It_formats_DateTimeOffset_correctly()
+            {
+                var date1 = DateTimeOffset.Now;
+
+                var html = date1.ToDisplayString("text/html");
+
+                html.Should().BeEquivalentHtmlTo(
+                    $"<span>{date1.ToString("u")}</span>");
+            }
+
+            [Fact]
+            public void It_formats_Enum_correctly()
```

Comment:
I would suggest `It_formats_enum_value_using_the_name` or something similarly more explicit about the intended outcome.

Reply:
Thanks done